tests/test_smoke.py

```diff
@@ -131,7 +131,7 @@ def test_minimal():
     test = Test(
         'minimal',
         [
-            f'sky launch -y -c {name} examples/minimal.yaml',
+            f'sky launch -y -c {name} --image-id sky:gpu-ubuntu-1804 examples/minimal.yaml',
             f'sky logs {name} 1 --status',  # Ensure the job succeeded.
             f'sky launch -y -c {name} examples/minimal.yaml',
```

Comment:
It will correctly error out as following:
```
Task from YAML spec: examples/minimal.yaml
sky.exceptions.ResourcesMismatchError: Requested resources do not match the existing cluster.
  Requested:    1x AWS(None, image_id='sky:gpu-ubuntu-2004') 
  Existing:     1x AWS(m6i.2xlarge, image_id='sky:gpu-ubuntu-1804')
To fix: specify a new cluster name, or down the existing cluster first: sky down test-image
```

Added a check for it.